multiview/cotraining/base.py

```diff
@@ -60,15 +64,17 @@ def __init__(self,
     def fit(self, Xs, y):
         """
         A method to co-trained estimators to multiview data.
+
         Parameters
         ----------
-        Xs: list of array-likes
+        Xs: numpy.ndarray, list of array-likes
```

Comment:
For clarity I would have an "or", not a comma

Reply:
Ok. I changed this in ctclassifier and base.py to match your updated docstring format